roles/kubernetes/control-plane/templates/kubeadm-config.v1beta4.yaml.j2

```diff
@@ -35,6 +35,10 @@ nodeRegistration:
 patches:
   directory: {{ kubeadm_patches_dir }}
 {% endif %}
+{% if kubeadm_custom_timeouts | length > 0 %}
```

Comment:
If we can render unconditionally (for instance empty dict when nothing custom), that might be better.